### PR TITLE
Nutmeg Upgrade - Code Merge 3/6

### PR DESCRIPTION
**Pull request by shadinaif**
## Change description

Nutmeg Upgrade - Code Merge 3/6. **Lilac Base** 😃 

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

Related to: https://appsembler.atlassian.net/browse/RED-3370

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: openedx/core/djangoapps/appsembler/sites/tests/test_site_delete_utils.py [hunk 1]
```diff
@@ -94,7 +94,6 @@ def test_get_models_using_course_key():
     assert len(classes) > 20, 'Should include a lot of models!'
 
     assert CourseOverview in classes, 'Should include CourseOverview'
-    assert AnonymousUserId in classes, 'Should include AnonymousUserId due to course_id field'
```
Comment (shadinaif) [inline openedx/core/djangoapps/appsembler/sites/tests/test_site_delete_utils.py:97]: Changed by edx from `CourseKeyField` to `LearningContextKeyField`

File: lms/djangoapps/grades/tests/test_tasks.py [hunk 2]
```diff
@@ -162,10 +162,10 @@ def test_block_structure_created_only_once(self):
             assert mock_block_structure_create.call_count == 1
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 1, 38, True),
-        (ModuleStoreEnum.Type.mongo, 1, 38, False),
-        (ModuleStoreEnum.Type.split, 3, 38, True),
-        (ModuleStoreEnum.Type.split, 3, 38, False),
```
Comment (OmarIthawi) [inline lms/djangoapps/grades/tests/test_tasks.py]: lovely 😅

**Review by OmarIthawi — APPROVED** [review]
Looks great! Thank you 💯 times.

File: cms/djangoapps/contentstore/management/commands/tests/test_migrate_transcripts.py [hunk 3]
```diff
@@ -10,7 +10,6 @@
 
 import ddt
```
[the remark this answers is not in the document]
Comment (shadinaif) [reply cms/djangoapps/contentstore/management/commands/tests/test_migrate_transcripts.py:11]: Noted 👍🏼 Thank you!

File: openedx/core/djangoapps/appsembler/sites/tests/test_site_delete_utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shadinaif) [reply openedx/core/djangoapps/appsembler/sites/tests/test_site_delete_utils.py:97]: Done

**Pull request merged**